tools/nymea-plugininfocompiler/nymea-plugininfocompiler.pro

```diff
@@ -4,14 +4,52 @@ CONFIG -= app_bundle
 
 include(../../nymea.pri)
 INCLUDEPATH += $$top_srcdir/libnymea $$top_builddir
-LIBS += -L$$top_builddir/libnymea -lnymea
+
+# FIXME: Rebuilding types here so that we can build the nymea-plugininfocompiler
+# without having to build the entire libnymea
+# Libnymea needs to be split up into 2 parts, one with full hardaremanager support
+# the other with something more minimalistic that doesn't pull in Bluetooth, ZigBee,
+# MQTT and whatnot.
 
 SOURCES += \
-        main.cpp \
-    plugininfocompiler.cpp
+    main.cpp \
+    plugininfocompiler.cpp \
+    $$top_srcdir/libnymea/integrations/pluginmetadata.cpp \
+    $$top_srcdir/libnymea/integrations/thingutils.cpp \
+    $$top_srcdir/libnymea/types/paramtype.cpp \
+    $$top_srcdir/libnymea/types/param.cpp \
+    $$top_srcdir/libnymea/types/vendor.cpp \
+    $$top_srcdir/libnymea/types/thingclass.cpp \
+    $$top_srcdir/libnymea/types/statetype.cpp \
+    $$top_srcdir/libnymea/types/eventtype.cpp \
+    $$top_srcdir/libnymea/types/actiontype.cpp \
+    $$top_srcdir/libnymea/types/interface.cpp \
+    $$top_srcdir/libnymea/types/interfacestatetype.cpp \
+    $$top_srcdir/libnymea/types/interfaceeventtype.cpp \
+    $$top_srcdir/libnymea/types/interfaceactiontype.cpp \
+    $$top_srcdir/libnymea/loggingcategories.cpp \
 
 HEADERS += \
-    plugininfocompiler.h
+    plugininfocompiler.h \
```

Comment:
`$$top_srcdir/` for completeness?

Reply:
erm. those are in the local dir tho... only the stuff that it needs to pull out from libnymea are with the absolute path